packages/rancher-pushprox/charts/values.yaml

```diff
@@ -60,6 +63,10 @@ clients:
     tag: v0.1.0-rancher1-client
   command: ["pushprox-client"]
 
+  copyCertsImage:
+    repository: rancher/rke-tools
```

Comment:
Is there a better Rancher image to use for this initContainer? All it does is run a simple Bash script does extremely basic operations (`echo`, `find`, `test`, `cp` and `chmod`) that can be performed by any lightweight image.

Reply:
I'd say rke-tools is too heavy-duty, and might have stuff on there we don't want in cluster from security perspective. We prepend busybox to our images.txt list, but I wonder if that will fail the list generation because its not in the rancher org. We have some checks around that. `rancher/kubectl` should work I think. But I like that idea that longterm we have a box that has those tools plus some networking ones like curl/wget/dig etc..

Reply:
Changed to `rancher/kubectl` in the latest commit!